packages/vscode-extension/src/accountTree.ts

```diff
@@ -2,33 +2,16 @@
   // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-import { AzureSolutionSettings, err, FxError, ok, Result, returnSystemError, returnUserError, SubscriptionInfo, TreeCategory, TreeItem, Void } from "@microsoft/teamsfx-api";
-import { AzureAccount } from "./commonlib/azure-account.api";
+import { AzureSolutionSettings, err, FxError, ok, Result, SubscriptionInfo, TreeCategory, TreeItem, Void } from "@microsoft/teamsfx-api";
 import AzureAccountManager from "./commonlib/azureLogin";
-import { ExtensionSource } from "./error";
-import { core, getSystemInputs, showError, tools } from "./handlers";
-import * as vscode from "vscode";
+import { core, getSystemInputs, tools } from "./handlers";
 import { askSubscription } from "@microsoft/teamsfx-core";
 import { VS_CODE_UI } from "./extension";
-
- 
-enum TelemetryTiggerFrom {
-  CommandPalette = "CommandPalette",
-  TreeView = "TreeView",
-}
-
-enum TelemetryProperty {
-  TriggerFrom = "trigger-from",
-}
-
-enum TelemetryEvent {
-  SelectSubscription = "select-subscription",
-}
-
-export enum AccountType {
-  M365 = "m365",
-  Azure = "azure",
-}
+import {
```

Comment:
Use definitions in extension to avoid duplication since it's moved to extension now